RNCryptor/RNDecryptor.m

```diff
@@ -198,13 +201,19 @@ - (void)addData:(NSData *)theData
     NSUInteger HMACLength = self.HMACLength;
     if (self.inData.length > HMACLength) {
       NSData *data = [self.inData _RNConsumeToIndex:self.inData.length - HMACLength];
-      [self decryptData:data];
+      if (data) {
+        [self decryptData:data];
+      }
```

Comment:
_RNConsumeToIndex: will never return nil, so this isn't needed.  More of a matter of how defensive one wants to be with a known private method.